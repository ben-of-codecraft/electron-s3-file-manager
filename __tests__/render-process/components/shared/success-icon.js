const React = require('react');
const ReactTestRenderer = require('react-test-renderer');
const SuccessIcon = require('../../../../src/renderer-process/components/shared/success-icon');

describe('success-icon', () => {
	test('render', () => {
		const render = ReactTestRenderer.create(<SuccessIcon/>);

		expect(render.toJSON()).toMatchSnapshot();
	});

	test('render with props', () => {
		const render = ReactTestRenderer.create(<SuccessIcon className="m-2"/>);

		expect(render.toJSON()).toMatchSnapshot();
	});
});
