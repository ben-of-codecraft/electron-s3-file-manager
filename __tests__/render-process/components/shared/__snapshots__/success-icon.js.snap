// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`success-icon render 1`] = `
<div
  className="swal2-icon swal2-success swal2-icon-show"
>
  <div
    className="swal2-success-circular-line-left"
  />
  <span
    className="swal2-success-line-tip"
  />
  <span
    className="swal2-success-line-long"
  />
  <div
    className="swal2-success-ring"
  />
  <div
    className="swal2-success-fix"
  />
  <div
    className="swal2-success-circular-line-right"
  />
</div>
`;

exports[`success-icon render with props 1`] = `
<div
  className="swal2-icon swal2-success swal2-icon-show m-2"
>
  <div
    className="swal2-success-circular-line-left"
  />
  <span
    className="swal2-success-line-tip"
  />
  <span
    className="swal2-success-line-long"
  />
  <div
    className="swal2-success-ring"
  />
  <div
    className="swal2-success-fix"
  />
  <div
    className="swal2-success-circular-line-right"
  />
</div>
`;
